crates/tiny/src/cmd.rs

```diff
@@ -159,6 +158,9 @@ fn close(args: CmdArgs) {
         MsgSource::User { serv, nick } => {
             ui.close_user_tab(&serv, &nick);
         }
+        // Don't allow closing the mentions tab
```

Comment:
@trevarj you disallowed closing mentions tab here.

Reply:
Are you ok with being able to close it?

Reply:
No, see my previous comment.